### PR TITLE
PLAN-600 add count and query to people

### PR DESCRIPTION
**Pull request by balen**

### CODE REVIEW

File: app/models/person.rb [hunk 1]
```diff
@@ -226,6 +232,24 @@ def assigment_consistency
...
+    schedule = PersonSchedule.arel_table
+
+    people_table.project(
+      people_table[:id].as('person_id'),
+      schedule[:person_id].count.as('session_count')
+    )
+    .join(schedule, Arel::Nodes::OuterJoin)
+    .on(
+      schedule[:person_id].eq(people_table[:id])
+      .and(schedule[:session_assignment_name].in(['Moderator', 'Participant']))
```
Comment (Gailbear) [inline app/models/person.rb:246]:
```suggestion
      .and(schedule[:session_assignment_name].in(['Moderator', 'Participant', 'Invisible']))
```

File: app/models/person.rb [hunk 2]
```diff
@@ -28,8 +28,14 @@ class Person < ApplicationRecord
 
   has_many  :session_limits
 
-  has_many  :person_schedules, dependent: :destroy
+  has_many  :person_schedules
   has_many  :person_schedule_approvals, dependent: :destroy
+  has_many  :scheduled_sessions,
+            -> {
+              where("person_schedules.start_time is not null and person_schedules.room_id is not null")
+              .where("person_schedules.session_assignment_name in (?)",['Moderator', 'Participant']) #, 'Invisible'
```
Comment (Gailbear) [inline app/models/person.rb:36]:
```suggestion
              .where("person_schedules.session_assignment_name in (?)",['Moderator', 'Participant', 'Invisible'])
```

**Review by Gailbear — CHANGES_REQUESTED** [review]
should include invisible, says the ticket. any idea why you didn't write it that way?

File: app/models/person.rb (hunk 1 above)
Comment (balen) [reply app/models/person.rb]: done

**Review by Gailbear — APPROVED** [review]

**Pull request merged**